src/libopenrave-core/jsonparser/jsonreader.cpp

```diff
@@ -211,7 +240,7 @@ class JSONReader
                 }
             }
 
-            boost::shared_ptr<const rapidjson::Document> prReferenceEnvInfo = _GetDocumentFromFilename(fullFilename, alloc);
+            boost::shared_ptr<const rapidjson::Document> prReferenceEnvInfo = _GetDocumentFromFilename(fullFilename);
```

Comment:
You cannot omit `alloc` here, revert the change, you have to pass the `alloc` in.